### PR TITLE
Add gamepad support

### PR DESCRIPTION
**Pull request by AustinHasten**

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.java [hunk 1]
```diff
@@ -385,6 +385,12 @@ public boolean onKeyUp(int keyCode, KeyEvent event) {
 	                return true;
 	            }
 	        }
+                if (keyCode == KeyEvent.KEYCODE_BUTTON_Y || keyCode == KeyEvent.KEYCODE_BUTTON_X
+                        || keyCode == KeyEvent.KEYCODE_BUTTON_B || keyCode == KeyEvent.KEYCODE_BUTTON_A)
+                {
+                    displayCardAnswer();
```
Comment (timrae) [inline AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.java]: this should only happen if `sDisplayAnswer` is False right?

Comment (AustinHasten) [reply AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.java]: Sorry about that, I foolishly put if after:

    if (sDisplayAnswer) {
    ....
    }

thinking that was sufficient to say the answer wasn't being shown. Obviously this is not the case. I'll fix it.

**Comment by AustinHasten** [comment]
Correct. If `sDisplayAnswer` is true, the buttons will always call `answerCard()` and the segment you referenced will never be reached. Would an explicit `else` statement be preferable though?

**Comment by timrae** [comment]
Yeah I think the readability is improved by being explicit in this case. Also can you please fix the indentation? It looks like there's one too many spaces in there.

**Comment by AustinHasten** [comment]
I put that segment in an `else` statement. I can't account for that extra space, it doesn't appear on my machine in Vim, Android Studio, or in my GitHub.

**Comment by timrae** [comment]
Thanks, it must just be the crappy github diff renderer then

**Pull request merged**